_puppeteers/tuxmea.md

```diff
@@ -6,16 +6,16 @@ bio: Automation is the key to success.
 twitter: tuxmea
 github: tuxmea
 linkedin: martinalfke
-forge: example42
+forge: nn
 slack: U11QGB2PN
 devto: tuxmea
 # You can find your StackExchange from the "network profile" link on your stackoverflow page
 stackexchange: # Use the number and username both from your url, eg `15186808/binford2k`
 ---
 
 Martin Alfke is working with Puppet since 2007. Since 2011 Martin conducts official Puppet Trainings in Germany.
-He is Co-Founder and CEO at [example42 GmbH](https://example42.com).
+He is CEO at [betadots GmbH](https://betadots.de).
```

Comment:
@tuxmea heads up that your URL does not resolve though it does with `www.` prepended.

Reply:
Thanks for checking.